Cargo.toml

```diff
@@ -21,43 +21,37 @@ exclude = [
 ]
 
 [dependencies]
-aead = "0.4"
-async-trait = "0.1"
-byteorder = "1.3"
-bytes = "1.0"
-chacha20poly1305 = "0.8"
-ed25519-dalek = { version = "1.0", features = ["std", "serde"] }
-erased-serde = "0.3"
+aead = "0.4.3"
+async-trait = "0.1.35"
+byteorder = "1.4.3"
+bytes = "1.1.0"
+chacha20poly1305 = "0.9.0"
+ed25519-dalek = { version = "1.0.1", features = ["std", "serde"] }
+erased-serde = "0.3.20"
 eui48 = { version = "1.0", features = ["serde"] }
 futures = "0.3"
 get_if_addrs = "0.5"
 hkdf = "0.11"
-hyper = { version = "0.14", features = ["server", "http1"] }
+hyper = { version = "0.14.18", features = ["server", "http1"] }
 libmdns = "0.6"
-log = "0.4"
-num = "0.2"
+log = "0.4.17"
+num = "0.4.0"
 rand = "0.7"
-serde = { version = "1.0", features = ["rc", "derive"] }
-serde_json = "1.0"
+serde = { version = "1.0.137", features = ["rc", "derive"] }
+serde_json = "1.0.81"
 sha2 = "0.9"
 signature = "1.1"
 srp = "0.5"
 thiserror = "1.0"
-tokio = "1.8"
-url = "2.1"
-uuid = { version = "0.8", features = ["v4", "serde"] }
+tokio = "1.18.1"
+url = "2.2.2"
+uuid = { version = "1.0.0", features = ["v4", "serde"] }
 x25519-dalek = "0.6"
 
-[build-dependencies]
-handlebars = "2.0"
-serde = { version = "1.0", features = ["derive"] }
-serde_json = "1.0"
-uuid = { version = "0.8", features = ["v4", "serde"] }
-
 [dev-dependencies]
-env_logger = "0.8"
-serde = { version = "1.0", features = ["derive"] }
-tokio = { version = "1.8", features = ["rt-multi-thread", "time", "macros"] }
+env_logger = "0.9.0"
+serde = { version = "1.0.137", features = ["derive"] }
+tokio = { version = "1.18.1", features = ["rt-multi-thread", "time", "macros"] }
```

Comment:
It's a matter of taste in the end, but I'd prefer keeping the patch semver blocks out of the `Cargo.toml`. Is there a reason you put them back in?